assets/fonts/fonts.css

```diff
@@ -1 +1,2 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter');
```

Comment:
Is it right way to import font or I need more options like for Roboto (:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap)?

Reply:
if we use the same service - then yes, we can put all fonts in one string, to avoid unnecessary requests.